sources/packages/backend/apps/workers/src/controllers/health-check/health.controller.ts

```diff
@@ -0,0 +1,27 @@
+import { Controller, Get } from "@nestjs/common";
+import {
+  HealthCheckService,
+  HealthCheck,
+  HealthCheckResult,
+} from "@nestjs/terminus";
+import { HealthService } from "@sims/services";
+
+@Controller("health")
```

Comment:
For workers, a `@Controller` indicates a class containing a job worker. Please add a comment explaining that this is an HTTP controllers intended only to be used to application healthy checks for liveness and readiness probes.